anyway/models.py

```diff
@@ -2055,3 +2056,56 @@ class EmbeddedReports(Base):
     report_name_english = Column(String(), primary_key=True)
     report_name_hebrew = Column(String())
     url = Column(String())
+
+
+class InfographicsDataCacheFields(object):
+    news_flash_id = Column(BigInteger(), primary_key=True)
+    years_ago = Column(Integer(), primary_key=True)
+    data = Column(sqlalchemy.types.JSON())
+
+
+class InfographicsDataCache(InfographicsDataCacheFields, Base):
+    __tablename__ = 'infographics_data_cache'
+    __table_args__ = (
+        Index('infographics_data_cache_id_years_idx', 'news_flash_id', 'years_ago', unique=True),
+
+    )
+
+
+    def get_data(self):
+        return self.data
+
+
+    def serialize(self):
+        return {
+            "news_flash_id": self.news_flash_id,
+            "years_ago": self.years_ago,
+            "data": self.data
+        }
+
+
+
+class InfographicsDataCacheTemp(InfographicsDataCacheFields, Base):
+    __tablename__ = 'infographics_data_cache_temp'
+
+
+    def serialize(self):
+        return {
+            "news_flash_id": self.news_flash_id,
+            "years_ago": self.years_ago,
+            "data": self.data
+        }
+
+    # Flask-Login integration
```

Comment:
not sure this part of 4 functions below is necessary. Did you just copy it from another place? Or do you use these functions?
@aviklai @idosavion @assafdayan any opinion on this one? When should we use [Flask-Login](https://flask-login.readthedocs.io/en/latest/)

Reply:
Perhaps @OriHoch will have an opinion here as well

Reply:
I saw it on other tables, and when things did not work I added it. Don't know whether it is actually needed.